src/main/resources/static/mastermind/mastermind.css

```diff
@@ -7,6 +7,11 @@ body {
     background-color: antiquewhite;
```

Comment:
Mozna pridat do css pro #comment-text display:block? To tlacitko jak je vedle toho mi prijde zvlastni, ale jen napad